src/components/benefit-cta-link.vue

```diff
@@ -64,10 +64,12 @@ export default {
   },
   methods: {
     getURL(link) {
+      if (!this.store.calculs.resultats._id) {
```

Comment:
Tu pourrais extraire `!this.store.calculs.resultats._id` dans une autre méthode afin de la nommer, p.e. `hasFinishedSimulation` ?